src/providers/krb5/krb5_child.c

```diff
@@ -3682,15 +3682,18 @@ static krb5_error_code privileged_krb5_setup(struct krb5_req *kr,
 
     if (!(offline ||
             (kr->fast_val == K5C_FAST_NEVER && kr->validate == false))) {
-        kerr = copy_keytab_into_memory(kr, kr->ctx, kr->keytab, &mem_keytab,
-                                       NULL);
-        if (kerr != 0) {
-            DEBUG(SSSDBG_OP_FAILURE, "copy_keytab_into_memory failed.\n");
-            return kerr;
-        }
+        /* A Keytab is not used if fast with anonymous pkinit is used (and validate is false)*/
+        if(!(kr->cli_opts->fast_use_anonymous_pkinit == true && kr->validate == false)){
```

Comment:
Please, add space after if and before { to make it compatible with our coding style. But I can do it prior merging it if you want me to.

Thank you.

Reply:
If you would be willing to add the spaces and adjust the spacing for the NULL in your comment below, please go ahead. If you would prefer I do it, I can.